Casks/font/font-g/font-goorm-sans.rb

```diff
@@ -0,0 +1,19 @@
+cask "font-goorm-sans" do
+  version "1.0.0"
+  sha256 "c59219d6ef005aa7035fa0ddd13612a36179bd59463b3f3b9ca960e44451dc52"
+
+  url "https://statics.goorm.io/fonts/goorm-sans/goorm-sans-#{version}.zip"
+  name "goorm Sans"
+  homepage "https://goorm-sans.goorm.io/"
+
+  livecheck do
+    url :homepage
+    regex(/href=.*?goorm-sans-(\d+(?:\.\d+)*)\.zip/i)
```

Comment:
```suggestion
    regex(/href=.*?goorm[._-]sans[._-]v?(\d+(?:\.\d+)+)\.zip/i)
```

We can be looser with delimiters, so changes to the naming scheme will not break our `livecheck` (see [documentation](https://docs.brew.sh/Brew-Livecheck#regex-guidelines)).
I added a similar suggestion to the other PR, too. Then we are good to go!

Reply:
Oh, I forgot to update this part. 😅 Thanks for pointing it out!